components/modules/lwip-base/lwip-init.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019, Data61
+ * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+ * ABN 41 687 119 230.
+ *
+ * This software may be distributed and modified according to the terms of
+ * the GNU General Public License version 2. Note that NO WARRANTY is provided.
+ * See "LICENSE_GPLv2.txt" for details.
+ *
+ * @TAG(DATA61_GPL)
+ */
+
+#include <autoconf.h>
+
+#include <string.h>
+#include <sel4/sel4.h>
+#include <platsupport/io.h>
+#include <sel4utils/sel4_zf_logif.h>
+#include <lwip/init.h>
+#include <lwip/dhcp.h>
+#include <lwip/ip_addr.h>
+#include <lwip/netif.h>
+#include <lwip/timeouts.h>
+#include <sel4bench/sel4bench.h>
+
+#define LWIP_TICK_MS 10
+
+bool timers_initialised;
```

Comment:
ensure this is properly initialized?
```suggestion
bool timers_initialised = false
```

Reply:
Unnecessary, as bools are initialised to false by default.